simple_network_sim/loaders.py

```diff
@@ -118,19 +187,23 @@ def _check_overlap(one, two):
 
 
 class AgeRange:
-    """A helper class for an age range."""
+    """A helper class for an age range.
+    
+    If b is None, it is assumed that a is a string to be
+    parsed.
```

Comment:
There's also the option of a being a tuple. Although, the feature is not currently used.

Reply:
Resolved by 6bb334f908d3966e1bedfd1279012636bf8ae132